### PR TITLE
OPENEUROPA-3048: Use the sparql_entity_storage module.

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,4 +122,32 @@ Or, if you can run commands on your host machine:
...
+* First deployment
+1. Install an empty version of the `drupal/sparql_entity_storage` module:
+   ```
+   $ composer require drupal/sparql_entity_storage:dev-empty-module
+   ```
+1. Deploy to production.
+1. Enable the module.
+
+* Second deployment
+1. Require `drupal/rdf_entity` with the new `1.0-alpha17` version and `drupal/rdf_skos` with the new `0.8.0` version.
```
Comment (sergepavle) [inline README.md:144]: 0.8.0 version or 0.9.0?

**Review by sergepavle — COMMENTED** [review]
I have left some question in PR.
Everything else looks good.

File: composer.json [hunk 2]
```diff
@@ -6,9 +6,9 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "drupal/rdf_entity": "1.0-alpha12",
+        "drupal/core": "^8.8",
         "easyrdf/easyrdf": "0.10.0-alpha.1 as 0.9.1",
+        "drupal/sparql_entity_storage": "^1.0.0-alpha5",
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply composer.json]: I have found that in the ewcms we already have enabled module (empty one). So looks like this PR related to second stage of deployments.

File: README.md [hunk 3]
```diff
@@ -122,4 +122,33 @@ Or, if you can run commands on your host machine:
...
+* First deployment
+1. Install an empty version of the `drupal/sparql_entity_storage` module:
+   ```
+   $ composer require drupal/sparql_entity_storage:dev-empty-module
+   ```
+1. Deploy to production.
+1. Enable the module.
+
+* Second deployment
+1. Remove the empty `drupal/sparql_entity_storage` module requirement
```
Comment (sergepavle) [inline README.md:144]:
```suggestion
1. Remove the empty `drupal/sparql_entity_storage` module requirement.
```

**Review by sergepavle — DISMISSED** [review]

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by nagyad — APPROVED** [review]

File: composer.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "prefer-stable": true,
     "require": {
         "drupal/core": "^8.8",
-        "drupal/rdf_entity": "1.0-alpha12",
+        "drupal/sparql_entity_storage": "^1.0.0-alpha6",
```
Comment (brummbar) [inline composer.json:10]: There's alpha8 now out, let's move to that.

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by brummbar — DISMISSED** [review]

**Review by brummbar — APPROVED** [review]

**Review by brummbar — DISMISSED** [review]

**Review by ademarco — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**